test/config/fep-config.toml.template

```diff
@@ -170,7 +170,7 @@ AggSenderPrivateKey = {Path = "{{.zkevm_l2_sequencer_keystore_file_path}}", Pass
 # LatestBlock, SafeBlock, PendingBlock, FinalizedBlock, EarliestBlock
 # Default value is "LatestBlock"
 # ------------------------------------------------------------------------------
-BlockFinality = "LatestBlock"
+#BlockFinality = "LatestBlock"
```

Comment:
As a side note, maybe we can remove this, AFAIK there is no `BlockFinality` in the `AggSender` config (anymore)?
```suggestion
```